test/land/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# test for creating pseudo ensemble for use in letkfoi
```

Comment:
This test does not seem to be actually created by cmake... I think I know why, we will discuss soon

Reply:
@jiaruidong2017 , you're missing ```add_subdirectory(land)``` in this [CMakeLists.txt](https://github.com/NOAA-EMC/GDASApp/blob/c26b3f58c6a49a4670ddebaf2229d9981671ffcf/test/CMakeLists.txt#L146)

Reply:
Thanks @guillaumevernieres